tests/scripts/all.sh

```diff
@@ -1277,6 +1277,7 @@ component_test_crypto_full_no_cipher () {
 component_test_crypto_default_stream_cipher_only () {
     msg "build: default with only stream cipher"
 
+    scripts/config.py crypto_full
```

Comment:
I think it should be `full` not `crypto_full`. The motivation for this is really to catch issues in TLS 1.2 code.

Reply:
I noticed that in full config tls 1.2 is disabled so it makes no sense to test it here instead I added psa and no psa configs.

Reply:
I think that's because you're using `crypto_full` not `full`. (I just tried, `full` leave TLS 1.2 enabled for me.) As the name implies, `crypto_full` only enables crypto (no X.509, no TLS).

That said, I'm happy with testing "default" and "default + use_psa" as well. But `crypto_full` was wrong.

Reply:
Yes, that is the case. I provided `crypto_full`  by mistake (copied wrong line) and treated as full. But if `default` with `psa/no psa` is sufficient I won't add full here.